### DISCUSSION TITLE
how do I enable the zero room without it messing up the growing of rooms in version 4.10.2

### DISCUSSION THREAD

**Discussion by TheBread1** [eccentricdevotion/TARDIS#636] [Q&A] [answered]
literally the title.. I've had to delete the tardis folder like 5 times, every time a failed attempt at enabling it without it breaking the growth of tardis rooms

**Comment by eccentricdevotion** [comment, marked as answer]
Same as previous versions - `/tardisconfig zero_room true` - then restart server

https://eccentricdevotion.github.io/TARDIS/zero-room.html

**Reply by TheBread1** [reply]
oh okay thank you lol, I thought it was tardisadmin.. evidently I'm crazy, thanks again!
